mirdata/datasets/mtg_jamendo_autotagging_moodtheme.py

```diff
@@ -0,0 +1,340 @@
+                m["TRACK_ID"]: {
+                    "ARTIST_ID": m["ARTIST_ID"],
+                    "ALBUM_ID": m["ALBUM_ID"],
+                    "PATH": m["PATH"],
+                    "DURATION": m["DURATION"],
+                    "TAGS": m["TAGS"],
+                }
+                for m in d[1:]
+            }
+
+        splits = []
+        for ii in range(5):
+            split = {}
+            path_train = os.path.join(
+                self.data_home,
+                "data",
+                "splits",
+                "split-" + str(ii),
+                "autotagging_moodtheme-train.tsv",
+            )
+            with open(path_train, "r") as fhandle:
+                d = list(
+                    csv.DictReader(
+                        fhandle,
+                        delimiter="\t",
+                        fieldnames=[
+                            "TRACK_ID",
+                            "ARTIST_ID",
+                            "ALBUM_ID",
+                            "PATH",
+                            "DURATION",
+                            "TAGS",
+                        ],
+                    )
+                )
+                split["train"] = [m["TRACK_ID"] for m in d[1:]]
+            path_validation = os.path.join(
+                self.data_home,
+                "data",
+                "splits",
+                "split-" + str(ii),
+                "autotagging_moodtheme-validation.tsv",
+            )
+            with open(path_validation, "r") as fhandle:
+                d = list(
+                    csv.DictReader(
+                        fhandle,
+                        delimiter="\t",
+                        fieldnames=[
+                            "TRACK_ID",
+                            "ARTIST_ID",
+                            "ALBUM_ID",
+                            "PATH",
+                            "DURATION",
+                            "TAGS",
+                        ],
+                    )
+                )
+                split["validation"] = [m["TRACK_ID"] for m in d[1:]]
+            path_test = os.path.join(
+                self.data_home,
+                "data",
+                "splits",
+                "split-" + str(ii),
+                "autotagging_moodtheme-test.tsv",
+            )
+            with open(path_test, "r") as fhandle:
+                d = list(
+                    csv.DictReader(
+                        fhandle,
+                        delimiter="\t",
+                        fieldnames=[
+                            "TRACK_ID",
+                            "ARTIST_ID",
+                            "ALBUM_ID",
+                            "PATH",
+                            "DURATION",
+                            "TAGS",
+                        ],
+                    )
+                )
+                split["test"] = [m["TRACK_ID"] for m in d[1:]]
+            splits.append(split)
+        return {"metadata": meta, "splits": splits}
```

Comment:
```suggestion
        meta["splits"] = splits
        return meta
```

This way the suggested change about `_track_metadata` above will work!